### PR TITLE
feat(942260): update .ra to utilise crs-toolchain formats, and rebuild

### PR DESCRIPTION
**Pull request by 53cur3M3**
fix 942260 update 942260.data to utilise crs-toolchain format

### CODE REVIEW

**Comment by 53cur3M3** [comment]
Should resolve https://github.com/coreruleset/coreruleset/issues/2860
Assembled rule required manual substition of :space: with \s\v , as :space: prevents apache starting.

File: util/regexp-assemble/data/942260.data [hunk 1]
```diff
@@ -3,18 +3,29 @@
 
 ##!+ i
 
+##!> template quotes [\"'`]
```
Comment (RedXanadu) [inline util/regexp-assemble/data/942260.data:6]: Is this template ever used?

File: regex-assembly/942260.ra [hunk 2]
```diff
@@ -3,18 +3,29 @@
 
 ##!+ i
 
+##!> define quotes [\"'`]
+
+
+##!> assemble
+  [\"'`]\s*?
```
Comment (theseion) [inline regex-assembly/942260.ra:10]:
```suggestion
  {{quotes}}\s*?
```

File: regex-assembly/942260.ra [hunk 3]
```diff
@@ -3,18 +3,29 @@
...
+  between
+  not
+  \|\|
+  &&
+  ##!=>
+  \s+[\s\w]+=\s*?\w+\s*?having\s+
+  ##!=>
+##!<
+
 \w\s+like\s+[\"'`]
```
Comment (theseion) [inline regex-assembly/942260.ra:28]:
```suggestion
\w\s+like\s+{{quotes}}
```

File: regex-assembly/942260.ra [hunk 4]
```diff
@@ -3,18 +3,29 @@
...
 like\s*?[\"'`]\%
```
Comment (theseion) [inline regex-assembly/942260.ra:29]:
```suggestion
like\s*?{{quotes}}\%
```

File: regex-assembly/942260.ra [hunk 5]
```diff
@@ -3,18 +3,29 @@
...
 [\"'`]\s*?like\W*?[\"'`\d]
```
Comment (theseion) [inline regex-assembly/942260.ra:30]:
```suggestion
{{quotes}}\s*?like\W*?[\"'`\d]
```

**Review by theseion — CHANGES_REQUESTED** [review]

**Comment by theseion** [comment]
I've taken over this PR and need someone else to do the review.

**Review by fzipi — APPROVED** [review]
LGTM

**Comment by dune73** [comment]
Thanks. Looks good for me too.

**Review by theseion — APPROVED** [review]

**Pull request merged**